packages/yarnpkg-core/package.json

```
@@ -37,6 +37,7 @@
     "@types/got": "^8.3.4",
     "@types/is-ci": "^2.0.0",
     "@types/micromatch": "^3.1.0",
+    "@types/node": "^13.7.6",
```

```suggestion
    "@types/node": "^13.7.0",
```
Should probably use the same version as the rest of the packages